[2022-03-13 22:08:48,221] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: 2_daily_transformation_analysis.daily_analysis scheduled__2020-08-01T00:00:00+00:00 [queued]>
[2022-03-13 22:08:48,233] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: 2_daily_transformation_analysis.daily_analysis scheduled__2020-08-01T00:00:00+00:00 [queued]>
[2022-03-13 22:08:48,234] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-13 22:08:48,234] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-13 22:08:48,234] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-13 22:08:48,256] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): daily_analysis> on 2020-08-01 00:00:00+00:00
[2022-03-13 22:08:48,262] {standard_task_runner.py:52} INFO - Started process 849 to run task
[2022-03-13 22:08:48,265] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '2_daily_transformation_analysis', 'daily_analysis', 'scheduled__2020-08-01T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/transform_and_analysis.py', '--cfg-path', '/tmp/tmp4smt3w0b', '--error-file', '/tmp/tmpb_piy29u']
[2022-03-13 22:08:48,265] {standard_task_runner.py:77} INFO - Job 46: Subtask daily_analysis
[2022-03-13 22:08:48,311] {logging_mixin.py:109} INFO - Running <TaskInstance: 2_daily_transformation_analysis.daily_analysis scheduled__2020-08-01T00:00:00+00:00 [running]> on host e176062f3fa2
[2022-03-13 22:08:48,418] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=2_daily_transformation_analysis
AIRFLOW_CTX_TASK_ID=daily_analysis
AIRFLOW_CTX_EXECUTION_DATE=2020-08-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-08-01T00:00:00+00:00
[2022-03-13 22:08:48,418] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-13 22:08:48,419] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'cd /dbt && dbt run --models analysis --profiles-dir .']
[2022-03-13 22:08:48,430] {subprocess.py:85} INFO - Output:
[2022-03-13 22:08:51,182] {subprocess.py:89} INFO - Running with dbt=0.20.0
[2022-03-13 22:08:53,972] {subprocess.py:89} INFO - Found 5 models, 0 tests, 0 snapshots, 0 analyses, 338 macros, 0 operations, 3 seed files, 0 sources, 0 exposures
[2022-03-13 22:08:53,973] {subprocess.py:89} INFO - 
[2022-03-13 22:08:55,957] {subprocess.py:89} INFO - 22:08:55 | Concurrency: 200 threads (target='dev')
[2022-03-13 22:08:55,957] {subprocess.py:89} INFO - 22:08:55 |
[2022-03-13 22:08:55,976] {subprocess.py:89} INFO - 22:08:55 | 1 of 2 START view model analysis.hotel_count_by_day.................. [RUN]
[2022-03-13 22:08:55,976] {subprocess.py:89} INFO - 22:08:55 | 2 of 2 START view model analysis.thirty_day_avg_cost................. [RUN]
[2022-03-13 22:08:57,151] {subprocess.py:89} INFO - 22:08:57 | 1 of 2 OK created view model analysis.hotel_count_by_day............. [[32mSUCCESS 1[0m in 1.17s]
[2022-03-13 22:08:57,281] {subprocess.py:89} INFO - 22:08:57 | 2 of 2 OK created view model analysis.thirty_day_avg_cost............ [[32mSUCCESS 1[0m in 1.30s]
[2022-03-13 22:08:57,794] {subprocess.py:89} INFO - 22:08:57 |
[2022-03-13 22:08:57,794] {subprocess.py:89} INFO - 22:08:57 | Finished running 2 view models in 3.82s.
[2022-03-13 22:08:57,810] {subprocess.py:89} INFO - 
[2022-03-13 22:08:57,811] {subprocess.py:89} INFO - [32mCompleted successfully[0m
[2022-03-13 22:08:57,811] {subprocess.py:89} INFO - 
[2022-03-13 22:08:57,811] {subprocess.py:89} INFO - Done. PASS=2 WARN=0 ERROR=0 SKIP=0 TOTAL=2
[2022-03-13 22:08:59,033] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-13 22:08:59,063] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=2_daily_transformation_analysis, task_id=daily_analysis, execution_date=20200801T000000, start_date=20220313T220848, end_date=20220313T220859
[2022-03-13 22:08:59,124] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-13 22:08:59,165] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
